project/Build.scala

```diff
@@ -155,8 +155,10 @@ object AlgebirdBuild extends Build {
   )
 
   lazy val algebirdTest = module("test").settings(
+    testOptions in Test ++= Seq(Tests.Argument(TestFrameworks.ScalaCheck, "-verbosity", "4"),
+                                Tests.Argument(TestFrameworks.ScalaCheck, "-minSuccessfulTests", "1")),
```

Comment:
If the scalacheck maintainer accepts my PR (https://github.com/rickynils/scalacheck/pull/188), then we'll be able to do the following:
- Ensure that ApproximateProperty props don't go in the same Properties class as normal props.
- For every Properties class that contains ApproximateProperty props, override the minSuccessfulTests to 1 (the default is 100).